catalog/imaging/catalog-imaging-transformer/src/main/java/com/connexta/alliance/transformer/nitf/NitfInputTransformer.java

```diff
@@ -59,7 +59,7 @@
 
     private static final String ID = "ddf/catalog/transformer/nitf";
 
-    private static final String MIME_TYPE = "image/nitf";
+    static final String MIME_TYPE = "image/nitf";
```

Comment:
I'd rather not do that now, since the change would go into ddf-2.10.0-SNAPSHOT and Alliance is set to ddf-2.9.0 release.